tests/cli/test_compose_alibaba.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Make sure we can build an image and deploy it inside Alibaba cloud!
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [ -z "$ALICLOUD_ACCESS_KEY" ]; then
+            rlFail "ALICLOUD_ACCESS_KEY is empty!"
+        else
+            rlLogInfo "ALICLOUD_ACCESS_KEY is configured"
+        fi
+
+        if [ -z "$ALICLOUD_SECRET_KEY" ]; then
+            rlFail "ALICLOUD_SECRET_KEY is empty!"
+        else
+            rlLogInfo "ALICLOUD_SECRET_KEY is configured"
+        fi
+
+        ALICLOUD_BUCKET="${ALICLOUD_BUCKET:-composer-test}"
+        ALICLOUD_REGION="${ALICLOUD_REGION:-us-east-1}"
+
+        rlLogInfo "ALICLOUD_BUCKET=$ALICLOUD_BUCKET"
+        rlLogInfo "ALICLOUD_REGION=$ALICLOUD_REGION"
+
+        for package in jq; do
+            if ! rlCheckRpm "$package"; then
+                rlRun -t -c "dnf -y install $package"
+                rlAssertRpm "$package"
+            fi
+        done
```

Comment:
Is the loop necessary here (same question as for `remove_old_objects_alibaba.sh`).